BlocSpot/app/src/main/java/com/izor066/android/blocspot/api/model/database/table/Table.java

```diff
@@ -0,0 +1,63 @@
+package com.izor066.android.blocspot.api.model.database.table;
+
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+/**
+ * Created by igor on 22/9/15.
+ */
+public abstract class Table {
+    public static interface Builder {
+
+        public long insert(SQLiteDatabase writableDB);
+    }
+
+    protected static final String COLUMN_ID ="id";
+
+    public abstract String getName();
+
+    public abstract String getCreateStatement();
+
+    public void onUpgrade(SQLiteDatabase writableDatabase, int oldVersion, int newVersion) {
```

Comment:
Do you plan to implement this method? If yes, a comment with a `//TODO implement upgrade path` (or something like this) would flag that this method is left un-implemented (yet) on purpose. Otherwise, if you're not sure, simply delete it until you need it. Leaving unimplemented methods in your code, especially when working within a team, can create a bit of confusion. 

[YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) is a good principle to follow in software development.